packages/cli-kit/assets/cli-ruby/lib/shopify_cli/constants.rb

```diff
@@ -67,7 +68,7 @@ module EnvironmentVariables
     module SupportedVersions
       module Ruby
         FROM = "2.7.5"
-        TO = "3.2.0"
+        TO = "3.2.10"
```

Comment:
Does this work? It would be nice to allow 3.2.11 without having to change this.

```suggestion
        TO = "~> 3.2"
```